torba/torba/client/basedatabase.py

```diff
@@ -480,14 +482,16 @@ def _transaction(conn: sqlite3.Connection, tx: BaseTransaction, address, txhash,
                 )
             txo = txs[row[0]].outputs[row[5]]
             txo.is_change = row[6] == 1
-            txo.is_my_account = row[7] == my_account
+            txo.is_my_account = row[7] in my_account_address
             txos.append(txo)
         return txos
 
     async def get_txo_count(self, **constraints):
         constraints.pop('offset', None)
         constraints.pop('limit', None)
         constraints.pop('order_by', None)
+        constraints.pop('my_accounts', None)
+        constraints['account'] = constraints.pop('filtering_accounts', constraints.pop('account', None))
```

Comment:
This looks wrong, is `account` a list or not? How can it hold both `filtering_accounts` which seems like a list and `account` which seems like a single account?

Reply:
Some of the transaction list command can only use `account` as they have not yet been changed to work with my and filtering account. Hence this was needed

Reply:
Also, `[account]` is not a single account. But changing it to `[accounts]` breaks SQL. So I think the case is that "account" is the name of a column in an SQL file which is used for filtering.

Reply:
Yes, looks like the `pubkey_address` table inside blockchain.db has a column names "account".